packages/react-vapor/src/components/table-hoc/TableRowConnected.tsx

```diff
@@ -146,7 +147,11 @@ class TableRowConnected extends React.PureComponent<
                 collapsibleRowToggle = React.isValidElement(customToggle) ? (
                     customToggle
                 ) : (
-                    <td className="table-row-collapsible-btn">
+                    <td
+                        className={classNames('table-row-collapsible-btn', {
+                            'mod-no-border-bottom': this.props.collapsible.noBorderBottom,

```

Comment:
What about the other cells of the row? should noBorderBottom prop also remove the border for all the other cells of the row?